helm/templates/inferenceservice.yaml

```diff
@@ -19,11 +19,11 @@ spec:
       name: ''
       resources:
         limits:
-          cpu: '2'
+          cpu: '8'
           memory: 8Gi
         requests:
-          cpu: '1'
-          memory: 4Gi
+          cpu: '8'
```

Comment:
I would set the CPU request to something lower. like `2`.  Because we don't need to "carve out this much. 
It's really the limit that needs to be at 8. 
Same comment for memory.

Reply:
Fair enough, updated to a lower request, thanks!